web-client/src/views/CaseDetail/DeleteCaseDeadlineModalDialog.tsx

```diff
@@ -23,12 +23,10 @@ export const DeleteCaseDeadlineModalDialog = connect(
         confirmSequence={confirmSequence}
         title="Are You Sure You Want to Delete This Deadline?"
       >
-        <div className="ustc-delete-case-deadline-modal margin-bottom-2">
-          <label className="margin-right-2" htmlFor="deadline-to-delete">
-            {form.month}/{form.day}/{form.year}
-          </label>
-          <span id="deadline-to-delete">{form.description}</span>
-        </div>
+        <label className="margin-right-2" htmlFor="deadline-to-delete">
+          {form.deadlineDate}
```

Comment:
Does this need to be formatted using `/` like previously?

Reply:
Good catch, thank you!